### PR TITLE
Release 0.11.0

### PR DESCRIPTION
**Pull request by ruiaraujo**
* Added `span` option where the tcp events will be logged. The interface matches the opentracing span interface.
* Improved retries performance and memory usage.

Blocked by #86

### CODE REVIEW

**Comment by oporkka** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
-## 0.10.0
+## 0.11.0
+
+* Added `span` option where the tcp events will be logged. The interface matches the opentracing span interface.
+* Improved retries performance and memory usage.
```
Comment (oporkka) [inline CHANGELOG.md]: Do we actually have some numbers for the retries claim?

Comment (ruiaraujo) [reply CHANGELOG.md]: No, but I am copying the claim of the circuit breaker changes which are similar. :smile:

**Comment by ruiaraujo** [comment]
:+1:

**Comment by oporkka** [comment]
:+1:

**Pull request merged**